examples/guestbook/frontend-service.yaml

```diff
@@ -11,5 +11,6 @@ spec:
   ports:
     # the port that this service should serve on
     - port: 80
+      targetPort: 80
```

Comment:
This is still work in progress. We are trying to get the release-1.1 branch green with our smoke tests without cherry-picking all 37 PRs since the fork from master. This very change is only for debugging the issue.